changelog.d/7230.feature

```diff
@@ -0,0 +1 @@
+Change defaults to require power level 100 for enabling encryption in a room.
```

Comment:
More of a nitpick here, so feel free to ignore, but something along the lines of "Change the default power level required to enable encryption in a room to 100." feels slightly simpler to read imho (and directly describes the consequence to the user).

Reply:
I might even go farther and not mention power levels, and just say "Admin" which users may be more familiar with. What do you think?

Reply:
wfm